aries_cloudagent/admin/server.py

```diff
@@ -323,7 +323,7 @@ async def check_token(request: web.Request, handler):
                 if (
                     valid_key
                     or is_unprotected_path(request.path)
-                    or (request.method == "OPTIONS")
+                    or (request.method in ["OPTIONS","HEAD"])
```

Comment:
Please revert this change.

Reply:
Reverted.  Put it for review because it was another approach, similar to OPTIONS.  Thanks for reviewing.